README.md

```diff
@@ -2,9 +2,11 @@
 [![npm](https://img.shields.io/npm/v/@blackbaud/sky-addin-client.svg)](https://www.npmjs.com/package/@blackbaud/sky-addin-client)
 [![status](https://travis-ci.org/blackbaud/sky-addin-client.svg?branch=master)](https://travis-ci.org/blackbaud/sky-addin-client)
 
-The SKY add-in client library facilitates creating custom add-ins to extend UI experiences within Blackbaud applications. There will be multiple types of add-ins (tiles, buttons, tabs, etc.) corresponding with various "extension points" throughout the system.  In each case, the general pattern is the same:  the add-in is registered by providing the URL that will be loaded in an iframe within the application.
+The SKY Add-in Client library facilitates creating custom add-ins to extend UI experiences within Blackbaud applications.  Developers can register the add-in URL with their SKY API application, and at runtime the add-in will be loaded into an iframe within the application.  This library must be used in the add-in for it to render within the Blackbaud application. The `AddinClient` class will integrate with the host page, passing data and commands between the host and the add-in's iframe.
```

Comment:
What's the correct capitalization for "SKY Add-in Client library?"  At https://github.com/blackbaud/skyux-lib-addin-client, we reference this as the "SKY Add-in Client Library."  Just looking for consistency, I don't know the right answer though.